CMakeLists.txt

```diff
@@ -113,4 +132,4 @@ target_include_directories(${TARGET_NAME} PRIVATE ${GSL_INCLUDES} "${PROJECT_SOU
 target_link_libraries(${TARGET_NAME} PUBLIC gsl)
 target_link_libraries(${TARGET_NAME} PUBLIC tables)
 
-install(TARGETS slim eidos DESTINATION bin)
+install(TARGETS slim eidos DESTINATION /home/users/ssriniv2/local/parallel_slim)
```

Comment:
obviously this is not the install destination that we want in GitHub :->

Reply:
Done!